controllers/apps/apimanager_controller.go

```diff
@@ -344,6 +344,23 @@ func (r *APIManagerReconciler) SetupWithManager(mgr ctrl.Manager) error {
 
 	resourceVersionChangePredicate := predicate.ResourceVersionChangedPredicate{}
 
+	cmEventMapper := &APIManagerConfigMapsEventMapper{
+		Context:   r.Context(),
+		K8sClient: r.Client(),
+		Logger:    r.Logger().WithName("APIManagerConfigMapsEventMapper"),
+	}
+
+	redisConfigLabelSelector := &apimachinerymetav1.LabelSelector{
+		MatchLabels: map[string]string{
+			"threescale_component":         "system",
```

Comment:
backend redis config map does not match this label selector. If it works now, please explain how.

Reply:
Maybe you can set two label selectors, one for system and another one for backend. Verify that works for either of them, that is like an `OR` operation.

Reply:
Test results for backend is available in Validation section , logs. But this is checked now :
```
[vmogilev@vmogilev 3scale-operator] (THREESCALE-9308-1)$ oc rsh backend-redis-f7b9658c8-qk52p
sh-5.1$ redis-cli
127.0.0.1:6379> replicaof
(error) ERR unknown command `replicaof`, with args beginning with: 
127.0.0.1:6379> slaveof
(error) ERR unknown command `slaveof`, with args beginning with: 
127.0.0.1:6379> 
sh-5.1$ 
exit
[vmogilev@vmogilev 3scale-operator] (THREESCALE-9308-1)$ date
Wed Apr  3 09:41:15 IDT 2024
```
I think it's working because we are checking `redis-config` CM , that used for both system and backend. 
```
$ oc describe cm redis-config
Name:         redis-config
Namespace:    3scale-test
Labels:       app=3scale-api-management
              threescale_component=system
              threescale_component_element=redis
Annotations:  <none>

Data
====
redis.conf:
----
.....
rename-command REPLICAOF ""
rename-command SLAVEOF ""

```

Reply:
Oh, I see https://github.com/3scale/3scale-operator/blame/master/pkg/3scale/amp/component/redis.go#L258

That is wrong :disappointed: . I guess it has been wrong since the refactor to use the same redis_reconciler and redis option provider for both redis instances, system and backend. 

Then, I would say the label selector only looks for ` threescale_component_element=redis`, remove the `threescale_component=system` because it is confusing and based on a mistake.

Reply:
Done. Thank you